internal/cli/login.go

```diff
@@ -187,14 +192,29 @@ func (o *LoginOptions) Run(ctx context.Context, args []string) error {
 		}
 
 		var err error
-		token, err = authProvider.Auth(o.Web, o.Username, o.Password)
+		authInfo, err = authProvider.Auth(o.Web, o.Username, o.Password)
 		if err != nil {
 			return err
 		}
+		token = authInfo.AccessToken
+		o.clientConfig.AuthInfo.RefreshToken = authInfo.RefreshToken
+		if authInfo.ExpiresIn != nil {
+			o.clientConfig.AuthInfo.AccessTokenExpiry = time.Now().Add(time.Duration(*authInfo.ExpiresIn) * time.Second).Format(time.RFC3339Nano)
+		}
+	} else {
+		o.clientConfig.AuthInfo.RefreshToken = o.RefreshToken
+		if o.ExpirationSeconds != 0 {
+			o.clientConfig.AuthInfo.AccessTokenExpiry = time.Now().Add(time.Duration(o.ExpirationSeconds) * time.Second).Format(time.RFC3339Nano)
+		}
 	}
 	if token == "" {
 		return fmt.Errorf("failed to retrieve auth token")
 	}
+	o.clientConfig.AuthInfo.AuthType = o.authConfig.AuthType
+	o.clientConfig.AuthInfo.AccessToken = token
+	o.clientConfig.AuthInfo.AuthCAFile = o.AuthCAFile
```

Comment:
I think it would be safer to store the content of the file. If user uses relative path, we will fail later on if `CLI` is called from different location.

Reply:
Easier solution will be to make sure that the path is absolute